packages/wonder-blocks-clickable/src/components/__docs__/accessibility.stories.mdx

```diff
@@ -0,0 +1,152 @@
+import {Meta, Story, Canvas} from "@storybook/addon-docs";
+import {StyleSheet} from "aphrodite";
+
+import Clickable from "@khanacademy/wonder-blocks-clickable";
+import Color from "@khanacademy/wonder-blocks-color";
+import {View} from "@khanacademy/wonder-blocks-core";
+import Icon, {icons} from "@khanacademy/wonder-blocks-icon";
+import Spacing from "@khanacademy/wonder-blocks-spacing";
+
+<Meta
+    title="Clickable / Clickable / Accessibility"
+    component={Clickable}
+    parameters={{
+        previewTabs: {
+            canvas: {hidden: true},
+        },
+        viewMode: "docs",
+        chromatic: {
+            // Disables chromatic testing for these stories.
+            disableSnapshot: true,
+        },
+    }}
+/>
+
+# Accessibility
+
+## Keyboard interactions
+
+| Key | Action |
+| --- | --- |
+| Enter or Space | Activates the clickable element |
+
+
+## Roles
+
+| Component | Role | Usage |
+| --- | --- | --- |
+| `<Clickable onClick={} />`  | button | A clickable button element |
+| `<Clickable href="/math" skipClientNav={true} />`  | link | A clickable anchor element |
+| `<Clickable href="/math" />`  | link | A clickable anchor element (using `react-router`'s Link) |
+
+## Attributes
+| Attribute | Usage |
+| --- | --- |
+| tabindex="0" | Includes the clickable element in the tab sequence. |
+| aria-disabled="true" | Indicates that the element is perceivable but disabled. |
+| aria-label="value" | Defines a string value that labels the clickable element. Use it in case the clickable element doesn't include any descriptive text (e.g. Icons, images) |
+
+## Examples
+
+### Labeling
+
+`Clickable` has an `ariaLabel` prop that sets the component's accessible name.
+ariaLabel should be passed when using graphical elements to let screen reader
+users know the purpose of the clickable element.
+
+_NOTE:_ If the clickable element is not graphicall, it's best to avoid using `ariaLabel` as the text content text content of the element itself, which is read by default, should ideally be descriptive enough to not need to manually pass in the label.
```

Comment:
```suggestion
_NOTE:_ If the clickable element is not graphical, it's best to avoid using `ariaLabel` as the text content of the element itself, which is read by default, should ideally be descriptive enough to not need to manually pass in the label.
```